pkgpanda/__init__.py

```diff
@@ -12,6 +12,7 @@
 try:
     import grp
 except ImportError:
+    # It is not possible to import grp on Windows.
```

Comment:
Thanks for adding an explanation. Even better, I think we should move this and `import pwd` below the import of `is_windows` and then replace the `try` with an `if not is_windows:`. This documents the case where it is not supported, but also errors if, for some reason, another platform does not have these modules available.

We can also get rid of the later `assert` checks that `pwd` and `grp` exist.

Reply:
Beautiful, I like it!